mailpoet/lib/Automation/Engine/Migrations/Migrator.php

```diff
@@ -47,6 +47,7 @@ public function createSchema(): void {
       CREATE TABLE {$this->prefix}workflow_runs (
         id int(11) unsigned NOT NULL AUTO_INCREMENT,
         workflow_id int(11) unsigned NOT NULL,
+        version_id int(11) unsigned NOT NULL,
```

Comment:
Do we now need to store both `version_id` and `workflow_id`? Or... should we?
I guess in the `WorkflowRun` we can have both, but on the DB level, I'm not sure if it's needed. On the other hand, there's no harm saving both... or is there? 🤔  What do you think?

Reply:
Technically speaking the `workflow_id` would not be necessary. You can join the version data base with the unique `version_id` and conclude what is the `workflow_id`. But to me, it is a different table. Maybe I just want quickly to fetch `Select workflow.id, workflow.name, count(workflow_runs.id) as num from workflow, workflow_runs where workflow_runs.workflow_id = workflow.id` or similar queries, where I really do not need data from the version table at all. This column would enable me to do so. Also, in debugging scenarios, I think it helps, to have quickly a high level information what workflow a run belongs to.

The harm I see is in more bytes stored, but I think it is convenient information to have in this table.

Reply:
@websupporter Makes sense! Let's keep it.